chart/compass/charts/prometheus-postgres-exporter/values.yaml

```diff
@@ -43,11 +43,6 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
-rbac:
-  # Specifies whether RBAC resources should be created
-  create: true
-  # Specifies whether a PodSecurityPolicy should be created
-  pspEnabled: true
```

Comment:
There is also an .md file referring to that value - `compass/chart/compass/charts/prometheus-postgres-exporter/README.md` we can remove it from there as well